### ISSUE TITLE
Discrepancy in returning SELECT results

### ISSUE THREAD

**Issue by abhikco** [SAP/node-hdb#9]
Hi,
For a JOIN query on two tables which have columns with the same name, a simple SELECT returns only the values of the matching columns in the second table.

Descriptive Example:
Table TAB1 (ID NUMBER, NAME NVARCHAR(512), AGE NUMBER)
Table TAB2 (NID NUMBER, NAME NVARCHAR(512), SALARY NUMBER)

Query: SELECT TOP 1 * FROM TAB1, TAB2 WHERE TAB1.ID = TAB2.NID;

Using node-hdb, the results for the query are of the format:
[{ID: 1, AGE: 55, NID: 1, NAME: 'ABC', SALARY: 35000}, {ID: 2, AGE: 55, NID: 2, NAME: 'BCD', SALARY: 350}, .... ]

In each object of the array, the value for the NAME field in the last table of the join is returned. The values of NAME for the first table (s) are lost. I understand this is inherent to Javascript object semantics that no object can have duplicate keys but I think it will be best to come up with a more streamlined way of returning back the results, especially in such cases where there's a potential for data 'loss'.

Best Regards,
Abhi

P.S.: Of course, the problem does not arise if one uses 'aliases' within the SELECT query for the columns. But this becomes cumbersome in most cases.

**Comment by holgerkoser** [comment]
This will be supported in the next release.
I will work like this:
```js
client.exec('select * from DUMMY', {
  nestTables: true
}, function (err, rows) {
  rows.should.eql([{
    DUMMY: {
        DUMMY: 'X'
    }
  }]);
});
```

**Comment by holgerkoser** [comment]
New query options `nestTables` is available with version 0.4.2.
